zipline/assets/_assets.pyx

```diff
@@ -43,25 +40,35 @@ cdef class Asset:
 
     cdef readonly object exchange
 
+    cdef readonly object currency
```

Comment:
What would you think about calling this field 'quote_currency' as it is shared across all Asset classes? I'm a little wary of simply 'currency' just because it may be confusing when working with the CurrencyPairs, but 'quote_currency' states the intended function across all Asset classes.